sources/packages/web/src/components/layouts/Institution/sidebar/ManageInstitutionSideBar.vue

```diff
@@ -94,6 +94,15 @@ export default defineComponent({
           },
         },
       });
+      sidebarBottomItems.value.push({
```

Comment:
I though that the menus were using `mdi` icons but as we can see in this file all icons are from `fa`. Please use the `fa` icon.